flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/plan/rules/physical/stream/StreamPhysicalPythonGroupWindowAggregateRule.java

```diff
@@ -78,11 +78,11 @@ public boolean matches(RelOptRuleCall call) {
                         .anyMatch(x -> PythonUtil.isPythonAggregate(x, PythonFunctionKind.PANDAS));
         boolean existJavaFunction =
                 aggCalls.stream().anyMatch(x -> !PythonUtil.isPythonAggregate(x, null));
+        if (existPandasFunction && existGeneralPythonFunction) {
+            throw new TableException(
```

Comment:
Also this might be your design decision before: why throw a exception than return false to indicate this rule does not match the case when both general python function and pandas functions appear together?

Reply:
I think the way of throwing exception can better make users know the specific cause of the error. What do you think?

Reply:
I see. 

This is ok as this is a relatively easy place to throw exception. I feel like conceptually the better place is in the validator but that might be too much work.